playground/public/assets/export-db.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+use Drupal\Core\DrupalKernel;
+use Symfony\Component\HttpFoundation\Request;
+
+$stdErr = fopen('php://stderr', 'w');
+
+set_error_handler(static function(...$args) use($stdErr) {
+    fwrite($stdErr, print_r($args,1));
+});
+
+$flavor = file_get_contents('/config/flavor.txt');
+unlink('/config/flavor.txt');
+
+$autoloader = require "/persist/$flavor/vendor/autoload.php";
+
+$request = Request::createFromGlobals();
+$kernel = DrupalKernel::createFromRequest($request, $autoloader, 'prod');
+chdir($kernel->getAppRoot());
+$kernel->boot();
+
+$sql = '';
```

Comment:
I think it needs to begin with `BEGIN TRANSACTION;` and end with `END TRANSACTION;`